### PR TITLE
dracut: add systemd-networkd to the initramfs.

### PR DESCRIPTION
**Pull request by vcaputo**

### CODE REVIEW

File: dracut/02systemd-networkd/module-setup.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This is derived from upstream dracut 043
+
+install() {
+    inst_multiple -o \
+        $systemdutildir/systemd-networkd \
+        $systemdutildir/systemd-networkd-wait-online \
+        $systemdsystemunitdir/systemd-networkd-wait-online.service \
```
Comment (marineam) [inline dracut/02systemd-networkd/module-setup.sh]: Isn't wait-online.target also neded?

Comment (marineam) [inline dracut/02systemd-networkd/module-setup.sh]: er, I mean network-online.target

Comment (marineam) [inline dracut/02systemd-networkd/module-setup.sh]: Oh, nm, those are in the systemd module

File: dracut/02systemd-networkd/module-setup.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This is derived from upstream dracut 043
```
Comment (marineam) [inline dracut/02systemd-networkd/module-setup.sh]: I didn't know that that had landed in dracut, does this differ in any significant way? If not we can just update dracut then.

Comment (marineam) [inline dracut/02systemd-networkd/module-setup.sh]: There was this but it is already outdated: https://github.com/coreos/portage-stable/pull/210

**Comment by marineam** [comment]
lgtm

**Comment by crawford** [comment]
LGTM

**Pull request merged**